pkg/reconciler/instances/istio/action.go

```diff
@@ -24,6 +24,36 @@ type ReconcileAction struct {
 	performer actions.IstioPerformer
 }
 
+func NewReconcileAction(performer actions.IstioPerformer) *ReconcileAction {
```

Comment:
Do we really need this constructor? If so, please add documentation. It is an exposed function.

Reply:
Yes, added small docu